File: TM 6 Exception Handling/Exception Handling/program2.py
# Tek Raj Joshi
# Superset ID: 1368453

def checkPrime(num):
    for i in range(2, num//2):
        if num % i == 0:
            return False
    return True

try:
    num = input("Enter the number: ")
    print(checkPrime(int(num)))
except Exception as e:
        print("Oops!", e, "occurred.")

File: TM 15 C#-Basics/C#-Basics/Arrays/program2.cs
// Tek Raj Joshi
// Superset ID: 1368453

using System;  
public class Program2  
{  
    public static void Main() 
{       
    int i,n,sum=0;
	Console.Write("Input the 10 numbers : \n"); 
	for (i=1; i<=10; i++)
	{
        Console.Write("Number-{0} :",i);
        n = Convert.ToInt32(Console.ReadLine());		
		sum += n;
	}
	Console.Write("The sum of 10 no is : ",sum);
   }
}